python/ql/test/experimental/dataflow/model-summaries/model_summaries.py

```diff
@@ -122,6 +122,21 @@ def explicit_identity(x):
 SINK(a)  # $ flow="SOURCE, l:-1 -> a"
 SINK(b)  # $ flow="SOURCE, l:-2 -> b"
 
+from foo import MS_Class
+
+c = MS_Class()
+a, b = c.instance_method(SOURCE)
+SINK_F(a)
+SINK(b)  # $ flow="SOURCE, l:-2 -> b"
+
+m = c.instance_method
+x, y = (SOURCE, NONSOURCE)
+SINK(x)  # $ flow="SOURCE, l:-1 -> x"
+SINK_F(y)
```

Comment:
you didn't use `m` here, which seems odd.

Reply:
Ha, indeed..